flink-runtime/src/test/java/org/apache/flink/runtime/operators/hash/MemoryHashTableTest.java

```diff
@@ -100,340 +101,371 @@
 	@Test
 	public void testDifferentProbers() {
 		final int NUM_MEM_PAGES = 32 * NUM_PAIRS / PAGE_SIZE;
-		
-		AbstractMutableHashTable<IntPair> table = new CompactingHashTable<IntPair>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE));
-		
+		testDifferentProbersCore(new CompactingHashTable<>(serializer, comparator, getMemory(NUM_MEM_PAGES, PAGE_SIZE)), NUM_MEM_PAGES);
```

Comment:
I can do that, but I would like to ask why is that important. I mean if one of these tests fail, then one of the lines can be simply commented out to see which hash table is making it fail.